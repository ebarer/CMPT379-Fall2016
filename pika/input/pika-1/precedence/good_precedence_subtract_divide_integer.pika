exec {

  print 57 - 82 / 43, _n_.
  print 13 - 1 / 19, _n_.
  print 16 - 68 / 38, _n_.
  print 45 - 68 / 40, _n_.
  print 36 - 54 / 66, _n_.
  print 77 - 59 / 8, _n_.
  print 2 - 42 / 70, _n_.
  print 95 - 45 / 51, _n_.
  print 23 - 58 / 71, _n_.
  print 93 - 89 / 69, _n_.
  print 62 - 97 / 3, _n_.
  print 86 - 32 / 16, _n_.

  print 3 / 55 - 45, _n_.
  print 63 / 14 - 99, _n_.
  print 5 / 50 - 11, _n_.
  print 23 / 59 - 79, _n_.
  print 14 / 16 - 9, _n_.
  print 63 / 45 - 64, _n_.
  print 44 / 6 - 9, _n_.
  print 33 / 6 - 99, _n_.
  print 15 / 47 - 9, _n_.
  print 9 / 77 - 72, _n_.
  print 24 / 23 - 10, _n_.
  print 17 / 87 - 68, _n_.

  print 65 - 15 / 48 - 99, _n_.
  print 67 - 10 / 28 - 93, _n_.
  print 2 - 79 / 38 - 65, _n_.
  print 76 - 86 / 74 - 55, _n_.
  print 38 - 8 / 28 - 18, _n_.
  print 78 - 10 / 10 - 28, _n_.
  print 26 - 24 / 38 - 68, _n_.
  print 56 - 92 / 78 - 67, _n_.
  print 35 - 12 / 13 - 59, _n_.
  print 24 - 6 / 23 - 30, _n_.
  print 99 - 22 / 3 - 33, _n_.
  print 16 - 31 / 62 - 37, _n_.

  print 8 / 71 - 85 / 99, _n_.
  print 98 / 77 - 46 / 27, _n_.
  print 57 / 4 - 93 / 59, _n_.
  print 38 / 51 - 92 / 92, _n_.
  print 13 / 89 - 24 / 30, _n_.
  print 97 / 68 - 59 / 73, _n_.
  print 10 / 8 - 59 / 47, _n_.
  print 98 / 8 - 79 / 78, _n_.
  print 26 / 4 - 10 / 30, _n_.
  print 34 / 7 - 68 / 53, _n_.
  print 51 / 43 - 64 / 91, _n_.
  print 30 / 43 - 41 / 9, _n_.

}

