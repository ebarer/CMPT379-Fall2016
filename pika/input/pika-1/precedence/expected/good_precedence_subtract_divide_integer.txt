56
13
15
44
36
70
2
95
23
92
30
84
-45
-95
-11
-79
-9
-63
-2
-94
-9
-72
-9
-68
-34
-26
-65
20
20
49
-42
-12
-24
-6
59
-21
0
0
13
-1
0
1
0
11
6
3
1
-4
